Automattic/jetpack
# CRM: Remove company name preview from onboarding wizard

## Proposed changes:

* This PR removes the company name suggestion from the onboarding wizard.
* I have left the field width much the same (50% instead of 48%) but this could also be changed.

### Other information:


## Jetpack product discussion

https://github.com/Automattic/zero-bs-crm/issues/2903

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

* Using a code snippets plugin (if testing this PR on a test site that has already run the wizard previously, such as a localhost site), add `update_option( 'zbs_wizard_run', 0 );` and load any admin page for it to run. Then open up the CRM dashboard (`wp-admin/admin.php?page=zerobscrm-dash`) which should load the initial onboarding wizard page.
* The company name suggestion saying 'Jetpack CRM' should now be gone.

Before:


<img width="873" alt="Screenshot 2023-02-27 at 9 24 48 am" src="https://user-images.githubusercontent.com/16754605/221524749-46a6a7f7-d91b-46c1-9eb2-74ae7730dda3.png">


After:


<img width="689" alt="Screenshot 2023-02-27 at 9 22 09 am" src="https://user-images.githubusercontent.com/16754605/221524535-634bd89a-e062-439d-9e1f-dd3fe10056eb.png">
